lib/common/grpc-service.js

```diff
@@ -457,11 +458,13 @@ GrpcService.createDeadline_ = function(timeout) {
 GrpcService.getError_ = function(err) {
   if (err && GRPC_ERROR_CODE_TO_HTTP[err.code]) {
     var defaultErrorDetails = GRPC_ERROR_CODE_TO_HTTP[err.code];
-    err.code = defaultErrorDetails.code;
-    err.message = err.message || defaultErrorDetails.message;
-    return err;
-  }
+    var errorObj = is.error(err) ? new Error() : {};
```

Comment:
Maybe we should rename `getError_` if we're not returning an error, since this gets an error or a status object?